common/utils/format-date.js

```diff
@@ -28,6 +28,13 @@ export function formatTime(date: Date): string {
   return london(date).format('HH:mm');
 }
 
+export function isTimePast(date: Date): boolean {
+  const momentNow = london();
+  const momentEnd = london(date);
+
+  return momentEnd.isBefore(momentNow);
+}
```

Comment:
Instead of using moment, I think we can just use operators on the `Date` object.
Not sure if we need the timezone stuff though...

Reply:
I believe we need the timezone stuff